docs/userguide/intro.md

```diff
@@ -64,6 +65,25 @@ learning how to manage data, volumes and mounts inside our containers.
 
 Go to [Managing Data in Containers](../tutorials/dockervolumes.md).
 
+## Managing Metadata (Labels) for Docker Objects
+
+_Labels_ are a mechanism for applying metadata to Docker objects, including:
+
+- Images
+- Containers
+- Local daemons
+- Volumes
+- Networks
+- Swarm nodes
+- Swarm services
+
+
+You can use labels to organize your images, record licensing information, annotate
+relationships between containers, volumes, and networks, or in any way that makes
+sense for your business or application.
+
+Go to [Managing Metadata \(Labels\) for Docker Objects](labels-custom-metadata.md).
```

Comment:
Good catch. I am not sure the parentheses needed escaping, but since () has a special meaning in Markdown links, I thought it might, and better safe than sorry. It's a moot point now.